TemplateMethod/src/main/java/devslp/designpatterns/TemplateMethod/SortStrategyExample/DoubleSortHandler.java

```diff
@@ -0,0 +1,29 @@
+package devslp.designpatterns.TemplateMethod.SortStrategyExample;
+
+public class DoubleSortHandler implements SortHandler {
```

Comment:
method Object [] getArray(); must be implemented as well as type in line 4 and cast in line 10 to (Double)